### PR TITLE
Update home descriptions and adds rule

### PR DESCRIPTION
**Pull request by mehmetipekpatates**
updates outdated stuff and epic wording, probably adds more typos that maintainers should edit out

### CODE REVIEW

**Comment by qwertyquerty** [comment]
You messed up this branch

**Comment by Crossedfall** [comment]
Since this includes the rule changes, this closes #5 and #6

File: app/templates/home.html [hunk 1]
```diff
@@ -12,11 +12,9 @@
     <h1 class="site-title">BeeStation, For Beginners</h1>
     <p class="text-indent">It appears you have stumbled into the cozy corner of the BeeStation website. It has the rules for the server, a server statistics page, a page to review server bans, a forum, and links to many more places you should visit!</p>
 
-    <p class="text-indent">BeeStation is a Space Station 13 server, hosted by Crossedfall, which was created in March 2018. We focus greatly on being newbie friendly. We are a Low RP server. Our codebase is /tg/, but we are constantly striving to change up our code and add in new features. We love to put a touch of fun into every round to keep things interesting.</p>
+    <p class="text-indent">BeeStation is an Space Station 13 server hosted by Crossedfall that came into being around March 2018.  We brandish ourselves as being a newbie friendly community with LRP and MRP variants, Golden and Sage respectively. Our server runs on modified /tg/ code.   </p>
```
Comment (Crossedfall) [inline app/templates/home.html]:
```suggestion
    <p class="text-indent">BeeStation is a Space Station 13 server, hosted by Crossedfall, which was created in March 2018.  We focus on being being a newbie friendly community, with LRP (Golden) and MRP (Sage) servers. Our codebase is built off of /tg/'s foundations, combining our own unique features with those found on other servers.</p>
```

This adds grammatical errors and doesn't read all that well.

File: app/templates/home.html [hunk 2]
```diff
@@ -12,11 +12,9 @@
...
 
-    <p class="text-indent">Our server is open source, so feel free to contribute to the code! We also have a Discord server with a great helpful community, a forum for ban appeals, staff applications, or other things, and an Informative Wiki.</p>
-
-    <p class="text-indent">While we are for beginners, we also specialize in Low RP chaotic fun! Hop on and enjoy the ride...</p>
+    <p class="text-indent">Our server is open source, so feel free to contribute to the code! We also have a Discord server with a great helpful community, a forum for ban appeals, staff applications, suggestions and discussions as well as an informative Wiki.</p>
```
Comment (Crossedfall) [inline app/templates/home.html]:
```suggestion
    <p class="text-indent">Our server is open source, so feel free to contribute to the code! We also have a Discord server with a great helpful community; a forum for ban appeals, staff applications, suggestions and discussions; as well as an informative Wiki.</p>
```

**Review by Crossedfall — CHANGES_REQUESTED** [review]
Minor grammar errors introduced, provided some suggestions to improve readability as well.

File: app/templates/home.html [hunk 3]
```diff
@@ -12,11 +12,9 @@
     <h1 class="site-title">BeeStation, For Beginners</h1>
     <p class="text-indent">It appears you have stumbled into the cozy corner of the BeeStation website. It has the rules for the server, a server statistics page, a page to review server bans, a forum, and links to many more places you should visit!</p>
 
-    <p class="text-indent">BeeStation is a Space Station 13 server, hosted by Crossedfall, which was created in March 2018. We focus greatly on being newbie friendly. We are a Low RP server. Our codebase is /tg/, but we are constantly striving to change up our code and add in new features. We love to put a touch of fun into every round to keep things interesting.</p>
+    <p class="text-indent">BeeStation is a Space Station 13 server, hosted by Crossedfall, which was created in March 2018.  We focus on being being a newbie friendly community, with LRP (Golden) and MRP (Sage) servers. Our codebase is built off of /tg/'s foundations, combining our own unique features with those found on other servers.</p>
```
Comment (Crossedfall) [inline app/templates/home.html]:
```suggestion
    <p class="text-indent">BeeStation is a Space Station 13 server, hosted by Crossedfall, which was created in March 2018.  We focus on being a newbie friendly community, with LRP (Golden) and MRP (Sage) servers. Our codebase is built off of /tg/'s foundations, combining our own unique features with those found on other servers.</p>
```

I made an oopsie in my corrections. Added an extra `being`. Sorry about that

File: app/config/rules/bs_golden.json [hunk 4]
```diff
@@ -2,6 +2,12 @@
 	"title": "BeeStation Rules",
 	"description": "",
 	"rules": [
+			"title": "Admin Discretion",
+			"clauses": [
+				"Admins have the authority to 'ignore' rules when they feel it's in the best interest of the server/playerbase. They will, however, be fully accountable for their actions.",
+				"Listen to them without argument during the round, but if you feel there is a problem with their actions, report them on the forums afterwards. The Admin Conduct can be found here: https://pastebin.com/dpY4CyJB"
```
Comment (Crossedfall) [inline app/config/rules/bs_golden.json]:
```
The Admin Conduct can be found here: https://pastebin.com/dpY4CyJB"
```
Missed this in my initial review. A pastebin link? Really? Can we not have a forum thread or something?

Comment (Crossedfall) [reply app/config/rules/bs_golden.json]: Edit and then link to this: https://wiki.beestation13.com/view/Admin_Conduct

**Comment by mehmetipekpatates** [comment]
ported bad pastebin to the wiki and changed the link in code

File: app/config/rules/bs_golden.json [hunk 5]
```diff
@@ -2,6 +2,12 @@
 	"title": "BeeStation Rules",
 	"description": "",
 	"rules": [
+			"title": "Admin Discretion",
```
Comment (qwertyquerty) [inline app/config/rules/bs_golden.json:6]: You have malformed json here, you're missing an opening curly bracket

File: app/config/rules/bs_sage.json [hunk 6]
```diff
@@ -2,6 +2,13 @@
 	"title": "RP Server Rules",
 	"description": "BeeRP is a medium roleplay server with higher/more roleplay being greatly encouraged. \"Medium\" is loosely seen in our server as having an environment in which you can roleplay a character yet still have room for shenanigans and humor. In other words, you should consider your character to be a separate entity from you as a player. Your character's actions and feelings in-game should be based solely on your character's experiences and not your own as the player. It is also encouraged to use role play dialogue before making split decisions such as using lethal force against another character",
 	"rules": [
+		{
+		"title": "Admin Discretion",
```
Comment (qwertyquerty) [inline app/config/rules/bs_sage.json]: Please keep consistent indentation

**Review by Crossedfall — APPROVED** [review]

**Pull request merged**